Makefile

```diff
@@ -0,0 +1,9 @@
+
+clean:
+	mvn clean
+
+package: clean
+	mvn -B package -DskipTests=true
+
+deploy: package
+	mvn -B deploy -DskipTests=true
```

Comment:
Don't we need separate steps for snapshot and release deployment? How would it know to which repository to deploy?

Reply:
> If your project.version contains SNAPSHOT (f.e., 1.2-SNAPSHOT) and you execute mvn deploy, artifacts will be deployed to your snapshot repository. If it doesn't (f.e., 1.2) - they will be deployed to your release repository.

https://stackoverflow.com/questions/9461225/how-to-differentiate-between-deploying-releases-vs-snapshots